package.json

```diff
@@ -9,31 +9,32 @@
   "scripts": {
     "analyze": "NODE_ENV=production webpack --config config/prod.webpack.config.js --env analyze=true",
     "build": "NODE_ENV=production webpack --config config/prod.webpack.config.js",
-    "coverage": "jest --coverage --runInBand --detectOpenHandles --forceExit",
```

Comment:
We've added these parameters specifically to avoid the tests hanging on openshift-ci. Removing them might reintroduce the issue.

Reply:
I can try adding them back, but when I try to run it locally with them it hangs on the very first test.